pkg/sdn/plugin/pod_linux.go

```diff
@@ -401,13 +408,27 @@ func (m *podManager) teardown(req *cniserver.PodRequest) error {
 		}
 	}
 
+	errList := []error{}
 	if netnsValid {
 		hostVethName, _, podIP, err := getVethInfo(req.Netns, podInterfaceName)
 		if err != nil {
 			return err
 		}
 
-		if err := m.ovs.TearDownPod(hostVethName, podIP); err != nil {
+		podSandboxID := ""
+		if podIP == "" {
+			filter := &kruntimeapi.PodSandboxFilter{
+				LabelSelector: map[string]string{
+					ktypes.KubernetesPodNamespaceLabel: req.PodNamespace,
+					ktypes.KubernetesPodNameLabel:      req.PodName,
+				},
+			}
+			if id, err := m.getPodSandboxID(filter, req); err == nil {
```

Comment:
req.containerId should already be the pod sandbox ID...  I'm pretty sure we don't need to re-request it, since at Teardown time the runtime is calling us directly with the sandboxID.